packages/enzyme/src/mountTracking.js

```diff
@@ -0,0 +1,31 @@
+import { get } from './configuration';
+
+let mountedWrappers = [];
+
+/**
+ * Stores a reference to a testing wrapper for later unmounting
+ * via unmountAllWrappers()
+ *
+ * @param {ReactWrapper|ShallowWrapper} wrapper
+ */
+export function trackMountedWrapper(wrapper) {
+  const { enableMountTracking } = get();
```

Comment:
i think this probably needs `options` to be passed through to `get`?

Reply:
I'm not sure I follow. `get` just returns a shallow copy of the configuration object, does it not?